alvr/server/cpp/alvr_server/alvr_server.cpp

```diff
@@ -215,31 +214,37 @@ void DeinitializeStreaming() {
     // nothing to do
 }
 
+void SendVSync(float frameIntervalS) {
+    vr::Compositor_FrameTiming timings = {sizeof(vr::Compositor_FrameTiming)};
+    vr::VRServerDriverHost()->GetFrameTimings(&timings, 1);
+
+    // Warning: if the vsync offset deviates too much from 0, the latency starts to increase.
+    vr::VRServerDriverHost()->VsyncEvent(-frameIntervalS * timings.m_nNumVSyncsReadyForUse);
```

Comment:
Is this the call you were talking about that doesn't exist on Linux / or work?

Reply:
Yes

Reply:
Should it be guarded then or is it even to keep it this way

Reply:
It's already guarded. https://github.com/alvr-org/ALVR/pull/1299/files/8b07089306929570a1294c6cec5b2bcd5c818b10#diff-d643c72c21f7572f3441d4b6559f400b415a1956fe072a19eb832ae17b8bace5R769